feature/experimental/bgp/ate_tests/base_bgp_session_parameters/README.md

```diff
@@ -0,0 +1,44 @@
+RT-1.1: Base BGP Session Parameters
+
+Summary
+
+    Base BGP session establishment, with required capabilities
+
+Procedure
+
+    Establish BGP session on ATE and DUT with table-based configuration to cover:
+        ○ Explicitly specified Router ID
+        ○ MD5 password
+        ○ Explicitly specified local and remote AS number
```

Comment:
I remember in a prior commit, you had both eBGP and iBGP cases. Maybe we could add them back like this?

* Explicitly specified different local and remote AS number (eBGP)
* Explicitly specified same local and remote AS number (iBGP)